apps/yggdrasil/lib/yggdrasil/role_resource.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Yggdrasil.RoleResource do
```

Comment:
No, I think that's fair every record in the table is a permission with a resource attached to it so I think permissions would be more dominant. I sort of just melded the permissions into the existing table, the name made more sense when it was just a role and resource.

I'm good with the changing it.